dockerfiles/splash-entrypoint.sh

```diff
@@ -13,7 +13,7 @@ splash_pid=""
 while true; do
   echo "Starting Splash..."
 
-  python3 \
+  timeout --signal=SIGKILL 180 python3 \
```

Comment:
Please add 180 as default value for environment variable and use it. So we could be able to parameterize the container if needed.